sdk/textanalytics/ai-text-analytics/samples-dev/analyzeSentiment.ts

```diff
@@ -2,8 +2,8 @@
 // Licensed under the MIT License.
 
 /**
- * Demonstrates how to analyze sentiment in documents.
- * An overall and per-sentence sentiment is returned.
+ * @summary analyzes the sentiment of a piece of text
```

Comment:
nit: does the summary need to be a proper sentence? (capitalized first latter, ends in a period)

Reply:
The reason we haven't required this beforehand is because these summaries are included verbatim in the READMEs, and we found that every sentence was of the form "This sample ${summary}."

Maybe it's a good opportunity to document all our samples a little more clearly. The best version of this, IMO, looks like:

```javascript
/**
 * In this sample, we use the sentiment analysis endpoint to retrieve estimations of sentiment (positive,
 * negative, or mixed) within some example text. The endpoint allows us to analyze sentiment on a per-sentence
 * or overall basis.
 *
 * @summary analyzes the sentiment of a piece of text
 */
```

Reply:
@xirzec, @deyaaeldeen : I made some changes to the doc comments on the top of the sample files. They're more verbose now, with the summary tag actually being a summary of the doc comment text.